api/proto/info/info.proto

```diff
@@ -493,6 +518,7 @@ message ZInfoDevice {
   repeated MaintenanceModeReason maintenance_mode_reasons = 46;
 
   string dormant_time = 47 [deprecated = true];
+  StorageInfo storage_info = 48;
```

Comment:
To plan for the future I would make this a "repeated" and then add a mountpoint string in StorageInfo.